gateway-service/src/main/java/org/zowe/apiml/gateway/filters/post/PostStoreLoadBalancerCacheFilter.java

```diff
@@ -81,4 +68,17 @@ public Object run() {
 
         return null;
     }
+
+    private boolean metadataExists(InstanceInfo selectedInstance) {
+        Map<String, String> metadata = selectedInstance.getMetadata();
+        return metadata != null;
+    }
+
+    private boolean lbTypeExistsAuthenticationExistsInstanceIdExists(InstanceInfo selectedInstance) {
```

Comment:
I think this would be more understandable as `lbTypeIsAuthentication` and move the `selectedInstance.getInstanceId() != null` check into `shouldFilter` directly.